meta-lxatac-software/recipes-core/images/lxatac-core-image-base.bb

```diff
@@ -3,6 +3,7 @@ SUMMARY = "LXA TAC image containing a selection of useful development tools"
 BAD_RECOMMENDATIONS = "rng-tools"
 
 IMAGE_FEATURES = "ssh-server-openssh empty-root-password"
+IMAGE_FEATURES += "tools-debug"
```

Comment:
I think either all of the `IMAGE_FEATURES` should be assigned at the same time or all should be `+=`-ed, but not half and half.

Reply:
Agreed and fixed.